docs/solutions/hypervisors/kvm/kvm-baremetal/variables.tf

```diff
@@ -0,0 +1,28 @@
+#Variables declaration file
+
+#OCI Environment variables
+variable "tenancy_ocid" {}
+
+variable "user_ocid" {}
+variable "fingerprint" {}
+variable "compartment_ocid" {}
+variable "region" {}
+variable "ssh_public_key_path" {}
+variable "ssh_private_key_path" {}
+
+#Template used variables
+variable "prefix" {}
+variable "availability_domain" {}
```

Comment:
Would be useful to set defaults for all variables where a default value can reasonably be used.